doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -519,6 +519,8 @@ Libraries for networking
     * Standardized encode and decode function names in the codec.
     * Moved the :c:func:`nrf_cloud_location_request_json_get` function from the :file:`nrf_cloud_location.h` file to :file:`nrf_cloud_codec.h`.
       The function is now renamed to :c:func:`nrf_cloud_location_request_msg_json_encode`.
+    * Allow only one file download at a time within the nrf_cloud library.
```

Comment:
```suggestion
    * Allow only one file download at a time within the library.
```
This is all about the nRF Cloud library anyway, so I think we don't need to have the library name here.